src/tile_cmd.h

```diff
@@ -131,8 +117,8 @@ typedef void AnimateTileProc(TileIndex tile);
 typedef void TileLoopProc(TileIndex tile);
 typedef void ChangeTileOwnerProc(TileIndex tile, Owner old_owner, Owner new_owner);
 
-/** @see VehicleEnterTileStatus to see what the return values mean */
-typedef VehicleEnterTileStatus VehicleEnterTileProc(Vehicle *v, TileIndex tile, int x, int y);
+/** @see VehicleEnterTileFlags to see what the return values mean */
```

Comment:
```suggestion
/** @see VehicleEnterTileState to see what the return values mean */
```
Or `States`, or remove the comment entirely. "See the return type what the return value means" is not exactly useful.